ZenPacks/zenoss/LinuxMonitor/migrate/DevCpuUtilizationGraph.py

```diff
@@ -0,0 +1,34 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2016, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+
+from Products.ZenModel.ZenPack import ZenPackMigration
+from Products.ZenModel.migrate.Migrate import Version
+
+import logging
+log = logging.getLogger("zen.migrate")
+
+
+class DevCpuUtilizationGraph(ZenPackMigration):
+
+    version = Version(2, 0, 0)
+
+    def migrate(self, pack):
+        log.info('Fixing the Device template')
+        try:
+            path = '/zport/dmd/Devices/Server/SSH/Linux/rrdTemplates/Device'
+            template = pack.dmd.unrestrictedTraverse(path)
+            graph = template.graphDefs._getOb('CPU Utilization')
+        except Exception:
+            log.info('Unable to find CPU Utilization graph for Device')
+        else:
+            template.graphDefs._delObject(graph.id)
+            log.info("Removing {0} graph from Device".format(graph.id))
```

Comment:
As a user I don't think this message seems kind of scary. Why would we remove the "CPU Utilization" graph from Device? What is Device? Something like the following might make more sense to the user.

"Removing redundant CPU Utilization graph."